composer.json

```diff
@@ -24,8 +24,8 @@
         "ext-soap": "*",
         "ext-curl": "*",
         "ass/xmlsecurity": "~1.0",
-        "symfony/framework-bundle": "~2.6",
-        "symfony/twig-bundle": "~2.6",
+        "symfony/framework-bundle": ">=2.8",
+        "symfony/twig-bundle": "~3.0",
```

Comment:
Use `^2.7 || ^3.0`. Otherwise you would support only Symfony 3.0, because the TwigBundle is part of `symfony/symfony`.